engine/src/main/java/io/zeebe/engine/util/SbeBufferWriterReader.java

```diff
@@ -85,7 +85,7 @@ public boolean tryWrap(DirectBuffer buffer, int offset, int length) {
   }
 
   public byte[] toBytes() {
-    final byte bytes[] = new byte[getLength()];
+    final byte[] bytes = new byte[getLength()];
```

Comment:
:upside_down_face:

Reply:
Yeah I tried the intellij refactoring :sweat_smile: